packages/mcp/README.md

```
@@ -234,6 +234,32 @@ Get a list of commits for a given repository.
+### ask_codebase
+
+Ask a natural language question about the codebase. This tool uses an AI agent to autonomously search code, read files, and find symbol references/definitions to answer your question. Returns a detailed answer in markdown format with code references, plus a link to view the full research session in the Sourcebot web UI.
+
+<details>
+<summary>Parameters</summary>
+
+| Name            | Required | Description                                                                                                                                    |
+|:----------------|:---------|:-----------------------------------------------------------------------------------------------------------------------------------------------|
+| `query`         | yes      | The query to ask about the codebase.                                                                                                           |
+| `repos`         | no       | The repositories that are accessible to the agent during the chat. If not provided, all repositories are accessible.                           |
+| `languageModel` | no       | The language model to use for answering the question. Object with `provider` and `model`. If not provided, defaults to the first model in the config. Use `list_language_models` to see available options. |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)